lale/sklearn_compat.py

```diff
@@ -49,9 +44,7 @@
 
 # This method (and the to_lale() method on the returned value)
 # are the only ones intended to be exported
-def make_sklearn_compat(
-    op: Union[Ops.Operator, "SKlearnCompatWrapper", Any]
-) -> "SKlearnCompatWrapper":
+def make_sklearn_compat(op):
```

Comment:
Can you please add a comment explaining that this function is only there for backward compatibility and will be removed in a future release? In fact, we may even want to declare it deprecated.

Reply:
done

Reply:
I also removed all calls to it from the lale code base